manager/models.py

```diff
@@ -595,7 +595,7 @@ class Recurs:
             (never, 'Never'),
             (daily, 'Daily'),
             (weekly, 'Weekly'),
-            (biweekly, 'Biweekly'),
+            # (biweekly, 'Biweekly'),
```

Comment:
Can this be removed altogether or does it need to be commented out?

Reply:
It can be, but I left it as more of a self-documenting comment, in case anyone in the future wonders why there is no rule with an index of 3.  (See line 593)